packages/api/src/index.ts

```diff
@@ -73,12 +73,28 @@ class Sdk {
           // cleans up any `operationId` that we might have into something that can be used as a
           // valid JS method.
           const operationId = next.getOperationId({ camelCase: true });
+          const originalOperationId = next.getOperationId();
 
-          return Object.assign(prev, {
+          const op = {
             [operationId]: ((operation: Operation, ...args: unknown[]) => {
               return core.fetchOperation(operation, ...args);
             }).bind(null, next),
-          });
+          };
+
+          if (operationId !== originalOperationId) {
+            // If we cleaned up their operation ID into a friendly method accessor (`findPetById`
+            // versus `find pet by id`) we should still let them use the non-friendly version if
+            // they want.
+            //
+            // This work is to maintain backwards compatibility with `api@4` and does not exist
+            // within our code generated SDKs -- those only allow the cleaner camelCase
+            // `operationId` to be used.
+            op[originalOperationId] = ((operation: Operation, ...args: unknown[]) => {
+              return core.fetchOperation(operation, ...args);
+            }).bind(null, next);
```

Comment:
wait so this code is only for the dynamic approach, correct?

Reply:
Correct `src/index` is the only part of this codebase that powers the dynamic approach.

Reply:
Here's the work in the codegen code where it uses the camelCase version: https://github.com/readmeio/api/blob/main/packages/api/src/cli/codegen/languages/typescript.ts#L743-L750